packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -419,14 +420,25 @@ export class DataStores implements IDisposable {
         );
     }
 
-    public async getDataStore(id: string, wait: boolean): Promise<FluidDataStoreContext> {
+    public async getDataStore(id: string, wait: boolean, viaHandle: boolean = false): Promise<FluidDataStoreContext> {
```

Comment:
Can we have a follow-up to make this required?

Reply:
Made it required